verification/curator-service/ui/src/components/UploadsTable.tsx

```diff
@@ -156,8 +156,21 @@ class UploadsTable extends React.Component<Props, UploadsTableState> {
                                 );
                                 response
                                     .then((result) => {
-                                        const flattenedSources = result.data.uploads.map(
-                                            (u) => {
+                                        const flattenedSources = result.data.uploads
```

Comment:
This change seems unrelated to this PR - the uploads table doesn't use the batchUpsert call. I also don't think this is the right way to achieve this outcome. It won't work correctly with paging e.g. if the server returns 50 uploads for the first page but we filter out 5 rows, it will show 45 rows in the current page but also have a next page button.

Perhaps remove this file from this PR? Then you or I can have a look at doing this on the server side.

Reply:
SG, I'll remove the file.